DuckDuckGo/TabViewController.swift

```diff
@@ -2370,7 +2370,7 @@ extension TabViewController: SecureVaultManagerDelegate {
                             withTrigger trigger: AutofillUserScript.GetTriggerType,
                             completionHandler: @escaping (SecureVaultModels.WebsiteAccount?) -> Void) {
   
-        if !isAutofillEnabled {
+        if !isAutofillEnabled, featureFlagger.isFeatureOn(.credentialsAutofill) {
```

Comment:
The naming of `isAutofillEnabled` threw me off here. To me, it sounds like it should be itself checking for a feature flag, but really it's just detecting whether the user has enabled Autofill in the settings. I don't consider it a blocker, but I'd be interested to see if we can make the naming more clear.